commit 503c9574c48a7fea7102a4563cd17a9cd3a7a1b3
Author: sveinung <sveinung@a0f10bec-cc02-0410-94fc-a9cfff90b4cd>
Date:   Wed Jul 13 01:46:02 2016 +0000

    actions: free the name act_prob
    
    ...so it can be used as a name for the action probability struct.
    
    See patch #7466
    
    git-svn-id: svn://svn.gna.org/svn/freeciv/trunk@33241 a0f10bec-cc02-0410-94fc-a9cfff90b4cd

diff --git a/client/gui-sdl2/action_dialog.c b/client/gui-sdl2/action_dialog.c
index 7be16d5..199a07a 100644
--- a/client/gui-sdl2/action_dialog.c
+++ b/client/gui-sdl2/action_dialog.c
@@ -1148,13 +1148,13 @@ static void action_entry(const enum gen_action act,
   possible).
 **************************************************************************/
 static const char *action_custom_text(const int action_id,
-                                      const action_probability act_prob,
+                                      const action_probability prob,
                                       const struct city *actor_homecity,
                                       const struct city *target_city)
 {
   static struct astring custom = ASTRING_INIT;
 
-  if (!action_prob_possible(act_prob)) {
+  if (!action_prob_possible(prob)) {
     /* No info since impossible. */
     return NULL;
   }
diff --git a/common/actions.c b/common/actions.c
index aa3e4c5..e79b358 100644
--- a/common/actions.c
+++ b/common/actions.c
@@ -615,24 +615,24 @@ const char *action_prepare_ui_name(int action_id, const char* mnemonic,
   Suitable for a tool tip for the button that starts it.
 **************************************************************************/
 const char *action_get_tool_tip(const int action_id,
-                                const action_probability act_prob)
+                                const action_probability prob)
 {
   static struct astring tool_tip = ASTRING_INIT;
 
-  if (action_prob_is_signal(act_prob)) {
-    if (action_prob_unknown(act_prob)) {
+  if (action_prob_is_signal(prob)) {
+    if (action_prob_unknown(prob)) {
       /* Missing in game knowledge. An in game action can change this. */
       astr_set(&tool_tip,
                _("Starting to do this may currently be impossible."));
     } else {
-      fc_assert(action_prob_not_impl(act_prob));
+      fc_assert(action_prob_not_impl(prob));
 
       /* Missing server support. No in game action will change this. */
       astr_clear(&tool_tip);
     }
   } else {
     /* The unit is 0.5% chance of success. */
-    const double converted = (double)act_prob / 2.0;
+    const double converted = (double)prob / 2.0;
 
     astr_set(&tool_tip, _("The probability of success is %.1f%%."),
              converted);
diff --git a/common/actions.h b/common/actions.h
index ceed12c..b7fb882 100644
--- a/common/actions.h
+++ b/common/actions.h
@@ -317,7 +317,7 @@ const char *action_prepare_ui_name(int action_id, const char* mnemonic,
                                    const action_probability prob,
                                    const char *custom);
 const char *action_get_tool_tip(const int action_id,
-                                const action_probability act_prob);
+                                const action_probability prob);
 
 struct action_enabler_list *
 action_enablers_for_action(enum gen_action action);
